pydoctor/templatewriter/pages/__init__.py

```diff
@@ -343,7 +432,8 @@ def baseTables(self, request, item):
         return [item.clone().fillSlots(
                           baseName=self.baseName(b),
                           baseTable=ChildTable(self.docgetter, self.ob,
-                                               sorted(attrs, key=lambda o:-o.privacyClass.value)))
+                                               sorted(attrs, key=lambda o:-o.privacyClass.value), 
+                                                ChildTable.lookup_loader(self.template_lookup)))
```

Comment:
The template lookup could be moved outside of the loop, to avoid doing redundant work.

Reply:
The `work` here is only call `TemplateLookup._templates.__getitem__` with `ChildTable.filename` that should be O(1).